docs/concepts/select.md

```diff
@@ -353,6 +353,69 @@ export class CityService {
 
 now we can use this `zooThemeParks` selector anywhere in our application.
 
+### Inheriting selectors  
+
+When we have states that share similar structure, we can extract the shared selectors into a base class which we can later extend from. If we have an `entities` field on multiple states, we can create a base class containing a dynamic `@Selector()` for that field, and extend from it on the `@State` classes like this.
+
+```TS
+export class EntitiesState {
+
+  static get entities() {
+    return createSelector([this], (state: { entities: any[] }) => {
```

Comment:
Will you do it generic? As we discussed previously `static entities<T>() { ... }`